analyzers/src/SonarAnalyzer.Common/Rules/ImplementSerializationMethodsCorrectlyBase.cs

```diff
@@ -29,29 +29,35 @@ namespace SonarAnalyzer.Rules
     public abstract class ImplementSerializationMethodsCorrectlyBase : SonarDiagnosticAnalyzer
     {
         protected const string DiagnosticId = "S3927";
+
+        private const string AttributeOnLocalMethodMessageFormat = "Serialization attributes on local methods are not considered.";
```

Comment:
```suggestion
        private const string AttributeOnLocalMethodMessageFormat = "Serialization attributes on local functions are not considered.";
```

Reply:
"local function" is the correct term
- https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/local-functions
- https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/local-function-attributes